### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MattCMK**
Added SDK commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+                        [OPTIONAL] Version of the model to run. Defaults to latest.
+  --provider_url_map PROVIDER_URL_MAP
+                        [OPTIONAL] JSON object of chain id to Web3 provider HTTP URL. Overrides settings in env vars.
+  --api_url API_URL     [OPTIONAL] Credmark API url. Defaults to the standard API gateway. You do not normally need to set this.
+  --api_url API_URL     [OPTIONAL] Credmark API url
+```
+
+To call any model we need to provide below parameters:
+
+- `-b` or `–block_number` : to define against which block number the model should run. If not provided it is set to latest block number available for the chain.
```
Comment (iwyrkore) [inline README.md:285]: block number is no longer required. If not specified, it uses the "latest" block from our ledger db.

File: README.md [hunk 2]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+                        [OPTIONAL] Input JSON or if value is "-" it will read input JSON from stdin.
+  -v MODEL_VERSION, --model_version MODEL_VERSION
+                        [OPTIONAL] Version of the model to run. Defaults to latest.
+  --provider_url_map PROVIDER_URL_MAP
+                        [OPTIONAL] JSON object of chain id to Web3 provider HTTP URL. Overrides settings in env vars.
+  --api_url API_URL     [OPTIONAL] Credmark API url. Defaults to the standard API gateway. You do not normally need to set this.
+  --api_url API_URL     [OPTIONAL] Credmark API url
+```
+
+To call any model we need to provide below parameters:
```
Comment (iwyrkore) [inline README.md:283]: Maybe change this to something about these are the important parameters (they're not necessarily required.)

File: README.md [hunk 3]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+
+- `-b` or `–block_number` : to define against which block number the model should run. If not provided it is set to latest block number available for the chain.
+- `-i` or `–input` : to provide input for a model in a predefined structure.(you can run command `credmark-dev list --manifests` to see the input format required for each model. See example below). If not provided it will default to “{}”.
```
Comment (iwyrkore) [inline README.md:286]: Input is no longer necessary if the model does not require input.

File: README.md [hunk 4]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+Example below shows simple output (list of all models and their version) of list command:
+```
+>credmark-dev list -h
+
+2022-03-04 11:42:46,691 - credmark.model.engine.model_loader - INFO - Loading manifest from model_paths: ['models']
+
+Loaded models:
+
+ - var: ['1.0']
+ - cmk-total-supply: ['1.0']
```
Comment (iwyrkore) [inline README.md:322]: I don't think we should list a lot of models here or people will think it's the full list. I think just show a few and a "..." gap.

File: README.md [hunk 5]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+
+# SDK Command Documentation 
+## Help command
+
+All the commands accept `-h` parameter for help, e.g.:
+
+```
+>credmark-dev -h
+
+usage: credmark-dev [-h] [--log_level LOG_LEVEL] [--model_path MODEL_PATH] [--manifest_file MANIFEST_FILE]
```
Comment (iwyrkore) [inline README.md:230]: This is an older version that doesn't have the new commands like "describe" and "models"

File: README.md [hunk 6]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+  --manifests
+  --json
+```
+Note: You can also run `list-models` command alternatively.
+
+Example below shows simple output (list of all models and their version) of list command:
+```
+>credmark-dev list -h
+
+2022-03-04 11:42:46,691 - credmark.model.engine.model_loader - INFO - Loading manifest from model_paths: ['models']
```
Comment (iwyrkore) [inline README.md:317]: Remove this logging line. It's just noise.

File: README.md [hunk 7]
```diff
@@ -218,6 +218,147 @@ You should create and keep your models under this folder. Note that we have appl
...
+- `-i` or `–input` : to provide input for a model in a predefined structure.(you can run command `credmark-dev list --manifests` to see the input format required for each model. See example below). If not provided it will default to “{}”.
+Model-slug: Name of the model (slug) to call the model.
+
+Note:  if chain ID is not mentioned explicitly in the parameter,  it defaults to 1. If the model is using web 3 instance then chain id (and blockchain) will be picked from the .env file we defined during setup (refer to “configure environment variable” section). If the model is using Credmark database then, by default, it will refer to the Ethereum blockchain.
+
+See the example below. Here, we are running the model “cmk-circulating-supply” at block_number 14000000.
+```
+>Credmark-dev run -b 14000000 cmk-circulating-supply -i "{}"
+
+2022-03-04 11:21:00,362 - credmark.model.engine.model_loader - INFO - Loading manifest from model_paths: ['models']
```
Comment (iwyrkore) [inline README.md:295]: Remove the log line

File: README.md (hunk 1 above)
Comment (MattCMK) [reply README.md:285]: done

File: README.md (hunk 6 above)
Comment (MattCMK) [reply README.md:317]: done

**Pull request merged**